### PR TITLE
Dev/new designs spike

### PR DESCRIPTION
**Pull request by batteries76**
Basic layout spike.
No Trello card for this one sorry - just wanted to try to smash out the new layout. Let me know if there are any changes and I'll try to push them through. 
Most of the work is in rejigging the `Layout` component to take the new progress dots and to be responsive. I made the call that the dots will be the same for the desktop and mobile unless we find some time at the end.
Some of the logic I feel could be improved, but I was a little backed into the corner with the header text coming and going in different circumstances.
If this is ok, then the good thing is that we can then just worry about the pages and the content in each as almost separate entities. 
Have had a chat with Meng tonight and he'll be having a crack at one page, and me at another. These will be the templates for the rest of the pages. Anyone else reading this is welcome to pick up any other page and have a go. I'll update Trello at some stage tomorrow.

### CODE REVIEW

File: components/Layout/Layout.jsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import Head from "next/head";
-import { Container, Flex, Text } from "@chakra-ui/react";
+import { Container, Flex, Text, Box } from "@chakra-ui/react";
+import { useRouter } from 'next/router'
```
Comment (hqtan) [inline components/Layout/Layout.jsx:3]: small nitpick :sweat_smile: 

```suggestion
import { useRouter } from 'next/router';
```

Comment (batteries76) [reply components/Layout/Layout.jsx:3]: Yes, we should settle on a linter perhaps. 
Will manually do these across the board now.

File: components/Layout/Layout.jsx [hunk 2]
```diff
@@ -13,15 +14,54 @@ export default function Layout({ children }) {
...
+  const router = useRouter()
+  return (
+    <Flex alignItems="center" flexGrow={1} justifyContent="space-around" height={ [titleBarHeightMob, titleBarHeightDesk] } top="0" w="100%" bg="#055E9E" color="white" zIndex={10}>
+      {
+        isText
+        ?
+          router.pathname === '/'
+          ?
+          <Text fontSize={["24px", "30px"]} fontWeight="bold" flex={1} textAlign="center">
+            SeeChange
```
[the remark this answers is not in the document]
Comment (batteries76) [reply components/Layout/Layout.jsx:39]: Very much agree. Was having a heap of moments when going through this. 
I think that should be a priority if there is space pre MVP (or after).

[the remark this answers is not in the document]
Comment (batteries76) [reply components/Layout/Layout.jsx:39]: That's an ugly section of code too. I'll have a think about refactoring, but will be moving on for now.

**Comment by hqtan** [comment]
fyi @batteries76 ,

- the `Layout` component is also used in the `pages/results.js` page, and since the `Layout` component now accepts more arguments, the `results` page is currently broken... are you able to address that, if the fix is a quick one?
- `yarn.lock` file is not needed, so I've removed the file; I'd prefer to stick with npm with production deployment in vercel (unless there's a good reason to move to yarn)

**Review by hqtan — APPROVED** [review]
fabulous

**Comment by batteries76** [comment]
Thanks @hqtan!

**Comment by hqtan** [comment]
[Spike to get the new designs into the 'layout'](https://trello.com/c/nDdQjW97/217-spike-to-get-the-new-designs-into-the-layout)

**Pull request merged**